### ISSUE TITLE
Add support for configuring exporter port

### ISSUE THREAD

**Issue by gberche-orange** [markormesher/iperf-prometheus-collector#41]
Thanks again for the awesome full rewrite ! Sorry I had forgot to submit this issue I had worked around by modifying the start command to the following 

```bash
              sed -i.back 's/9030/{{ $fromAzPortNumber }}/g' ./build/server.js;
              yarn start
```

Can not apply the same workaround anymore with statically compiled go binary

https://github.com/markormesher/iperf-prometheus-collector/blob/489b51297f78311e117af7622f1e4989b0cc78d0/cmd/main.go#L27

Rationale: 

As a user,
In order to run multiple instance of the exporter with different options/targets on a given host,
I need to choose a distinct listening port for each exporter instance

**Comment by markormesher** [comment]
Are you running this in the container? If so you should be able to map it to whichever port you want to run it on. I'll add an option for the port either way though.

**Comment by markormesher** [comment]
Changes included on v0.3.0-rc3
